docs/TELEMETRY_SPECIFICATION.md

```
@@ -0,0 +1,1167 @@
+| `net.peer.ip` | string | "192.168.1.100" |
+
+**Context Extraction**: Extract `traceparent`/`tracestate` from HTTP headers.
+
+#### 2.2.2 HTTP Client (CLIENT span)
+
+**When**: Making HTTP request to agent
+
+**Span Name**: `HTTP {METHOD}` or `HTTP {METHOD} {TARGET_SERVICE}`
+
+**Required Attributes**:
+| Attribute | Type | Example |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)